src/fury-emitter.js

```diff
@@ -2,6 +2,7 @@ var events = require('events');
 
 // Default logging function
 function log(message) {
+  /*eslint no-unused-vars: 0 */
```

Comment:
This just tells the linter to ignore unused variables in that function, i.e. do not fail the build. We want to keep the unused variable as an example to others writing their own loggers.